slash/react/src/Form/Radio/__tests__/Radio.test.tsx

```diff
@@ -0,0 +1,71 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { describe, it, expect, vi } from "vitest";
+
+import { Radio, RadioModes } from "../Radio";
+
+const options = [
+  { value: "option1", label: "Option 1" },
+  { value: "option2", label: "Option 2", disabled: true },
+  { value: "option3", label: "Option 3" },
+];
+
+describe("Composant Radio", () => {
+  it("affiche toutes les options avec le bon label", () => {
+    render(
+      <Radio options={options} value="option1" mode={RadioModes.default} />,
+    );
+    options.forEach((option) => {
+      expect(screen.getByLabelText(option.label)).toBeInTheDocument();
+    });
+  });
+
+  it("coche l’option correspondant à la valeur passée", () => {
+    render(<Radio options={options} value="option1" />);
+    const radio1 = screen.getByLabelText("Option 1") as HTMLInputElement;
```

Comment:
si tu utilise le getByRole tu n'arrais pas besoin de caster en HTMLInputElement

```suggestion
    const radio1 = screen.getByRole("radio", { name: "Option 1" });
```